programs/address-lookup-table/src/lib.rs

```diff
@@ -13,3 +15,14 @@ pub use solana_sdk::address_lookup_table::{
     program::{check_id, id, ID},
     state,
 };
+
+#[cfg(target_os = "solana")]
+#[deprecated(
+    since = "1.17.0",
+    note = "Please use `solana_program::address_lookup_table` instead"
+)]
```

Comment:
The `#[deprecated]` attribute doesn't seem to be doing anything for `use` statements as I don't get any warnings about depreciation for these re-exports. Still, it could be useful to see it when looking at the source code.